hack/preload-images/preload_images.go

```diff
@@ -113,6 +118,26 @@ func verifyDockerStorage() error {
 	return nil
 }
 
+func verifyPodmanStorage() error {
+	cmd := exec.Command("sudo", "podman", "info", "-f", "json")
```

Comment:
is this expecting the system running the preload generation script to have installed podman ? or did you mean to run this in the podman inside minikube /
exec.Command("docker", "exec", profile, "sudo", "podman", "info", "-f", "json")

Reply:
I just copied the docker version, it seems both of them are wrong

Reply:
Yeah you are right both of them are wrong let's fix both of them

Reply:
Added the prefix, but also needed to move the verification to after creation...